final_project/pages/ProfilePage.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from "playwright/test";
+import { Base } from "./Base";
+import { locators } from "../data/locators";
+import { keyWords } from "../data/keywords";
+import { pageUrls } from "../data/pageurls";
+import { waitForPageLoadAndElVisible } from "../waiters/waiterBeforeEls";
+
+export class ProfilePage extends Base {
+    async getProfileDetail(fieldName: string) {
+        await expect(this.page.locator(locators.profilePage.field(fieldName))).toBeVisible();
+    }
+
+    async editProfileDetail(fieldName: string, value: string) {
```

Comment:
async editProfileDetail(profileData) {
        if(profileData.name) {
            await this.page.fill(locators.profilePage.inputField(name), profileData.name);
       }
      if(profileData.lastName) {
            await this.page.fill(locators.profilePage.inputField(lastName), profileData.lastName);
       } 
     clickUpdateButton()
    }

Reply:
editProfileDetail({name: 'Dasha', lastName: 'Ivanova', age: 33})